README.md

```diff
@@ -101,7 +103,6 @@ tape ansible deploy -l staging
 ## Testing
```

Comment:
Contributing.md is typically reserved for guidelines on how to submit PRs and Issues.